ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -66,37 +66,44 @@ podman pull quay.io/quads/badfish:latest
 USER=$(curl -sSk $QUADS_INSTANCE | jq -r ".nodes[0].pm_user")
 PWD=$(curl -sSk $QUADS_INSTANCE  | jq -r ".nodes[0].pm_password")
 if [[ "$TYPE" == "mno" ]]; then
-  HOSTS=$(curl -sSk $QUADS_INSTANCE | jq -r ".nodes[1:4+"$NUM_WORKER_NODES"][].pm_addr")
+  HOSTS=$(curl -sSk $QUADS_INSTANCE | jq -r ".nodes[1:4+"$NUM_WORKER_NODES"][].name")
 elif [[ "$TYPE" == "sno" ]]; then
-  HOSTS=$(curl -sSk $QUADS_INSTANCE | jq -r ".nodes[1:1+"$NUM_SNO_NODES"][].pm_addr")
+  HOSTS=$(curl -sSk $QUADS_INSTANCE | jq -r ".nodes[1:1+"$NUM_SNO_NODES"][].name")
 fi
 echo "Hosts to be prepared: $HOSTS"
 # IDRAC reset
 if [[ "$PRE_RESET_IDRAC" == "true" ]]; then
   echo "Resetting IDRACs ..."
   for i in $HOSTS; do
     echo "Resetting IDRAC of server $i ..."
-    podman run quay.io/quads/badfish:latest -v -H $i -u $USER -p $PWD --racreset
+    podman run quay.io/quads/badfish:latest -v -H mgmt-$i -u $USER -p $PWD --racreset
   done
   for i in $HOSTS; do
-    if ! podman run quay.io/quads/badfish -H $i -u $USER -p $PWD --power-state; then
+    if ! podman run quay.io/quads/badfish -H mgmt-$i -u $USER -p $PWD --power-state; then
       echo "$i iDRAC is still rebooting"
       continue
     fi
   done
 fi
+if [[ "$PRE_PXE_LOADER" == "true" ]]; then
+  echo "Modifying PXE loaders ..."
+  for i in $HOSTS; do
+    echo "Modifying PXE loader of server $i ..."
+    hammer -u $LAB_CLOUD -p $PWD host update --name $i --operatingsystem "RHEL 8.6" --pxe-loader "PXELinux BIOS" --build 1
```

Comment:
do we want to template out the osname? Maybe a job would want to override this?

Reply:
Not here, here the OSNAME is not relevant, we just need the servers to come back from UEFI PXE to this one.
The OSNAME var is present on the Foreman step to make sure the user is able to choose the OS to deploy.

Reply:
Adjusted! Included a new `FOREMAN_OS` var